_security/configuration/configuration.md

```diff
@@ -162,65 +163,4 @@ To learn about configuring the authentication backends, see the [Authentication
 * [Active Directory and LDAP]({{site.url}}{{site.baseurl}}/security/authentication-backends/ldap/)
 * [Proxy-based authentication]({{site.url}}{{site.baseurl}}/security/authentication-backends/proxy/)
 * [Client certificate authentication]({{site.url}}{{site.baseurl}}/security/authentication-backends/client-auth/)
-
-
-<!--- Remvoving Kerberos documentation until issue #907 is resolved.
```

Comment:
I see that https://github.com/opensearch-project/security-dashboards-plugin/issues/907 is still open. While we don't have automated functional tests in the security-dashboards-plugin to verify kerberos, has it been validated through a demo configuration?

Reply:
I assume that kerberos is still not supported on OpenSearch Dashboards, but this documentation is rather about OpenSearch, where kerberos is supported. We've been using it successfully on all our clusters already from the OpenDistro era.